package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "buttercup-website",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

Comment:
This is handed during please. Please revert it.

Also, you might revert the changes to package-lock as it has a ton of changed lines.

Reply:
Ah, I wasn't sure how it was handled. OK